### PR TITLE
DOP-2578: update cards to use new typography

### PR DESCRIPTION
**Pull request by seungpark**
### Stories/Links:

[DOP-2578](https://jira.mongodb.org/browse/DOP-2578)

### Current Behavior:

[Guides](https://www.mongodb.com/docs/guides/)
[Landing](https://www.mongodb.com/docs/)
[Drivers](https://www.mongodb.com/docs/drivers/)

### Staging Links:
[Guides](https://docs-mongodbcom-integration.corp.mongodb.com/098a51155afa851ada950d2f32e93107a46bcd59/master/guides/seung.park/DOP-2578-post-typography/)
[Landing](https://docs-mongodbcom-integration.corp.mongodb.com/ca1cc8487251860566cc18198a0059a91c4d4fd7/master/landing/seung.park/DOP-2578-post-typography/)
[Drivers](https://docs-mongodbcom-integration.corp.mongodb.com/098a51155afa851ada950d2f32e93107a46bcd59/master/drivers/seung.park/DOP-2578-post-typography/)

### Notes:
- used updated typography to fill in the card content, applying when necessary. Cards, Driver Index Tiles, Chapters, and FeedbackTab were the only ones I found in use
- operation and quiz components were not updated due to lack of content from docs repos? cmiiw but these seem unused.
- seeing a momentary flash of clashing CSS styles on landing and drivers pages. investigation in progress

### CODE REVIEW

File: src/components/DriversIndexTiles.js [hunk 1]
```diff
@@ -130,10 +137,12 @@ const DriversIndexTiles = () => {
...
     <StyledGrid>
       {tiles.map((t) => (
-        <StyledCard as={Link} contentStyle="clickable" key={t.title} to={t.slug}>
-          <StyledIcon>{t.icon}</StyledIcon>
-          {t.title}
-        </StyledCard>
+        <GatsbyLink key={t.title} to={t.slug}>
+          <StyledCard className="styled-card-from-driver-index-tiles">
+            <StyledIcon>{t.icon}</StyledIcon>
+            <Link to={t.slug}>{t.title}</Link>
```
Comment (gjchong25) [inline src/components/DriversIndexTiles.js:143]: Couldn't find the designs for the Drivers cards specifically, but should there be a link inside of a link? Looking at production it seems like the internal text isn't a link so I was wondering if this is intentional.

**Review by gjchong25 — COMMENTED** [review]
Screen Shot 2022-08-18 at 10 40 14 AM
Generally the cards look good, just had a few questions and comments. I did have one note about this screenshot: should this "Still Learning MongoDB" card that shows when you scroll down the page also be styled similarly to the other Cards? Currently it doesn't have the rounded borders like the new Cards.

File: src/components/DriversIndexTiles.js (hunk 1 above)
Comment (seungpark) [reply src/components/DriversIndexTiles.js:143]: this was due to designs here https://www.figma.com/file/PctpWXNmlv0khZahanOsU7/FY22-New-Brand?node-id=137%3A9885

the tiled cards showed a blue link line, but after discussion with @allisonmui9 , i think its safe to leave as is, as a non-link

File: src/components/Card/index.js [hunk 2]
```diff
@@ -74,6 +70,16 @@ const CompactTextWrapper = styled('div')`
   @media ${theme.screenSize.upToSmall} {
     margin-left: ${theme.size.default};
   }
+
+  p {
```
[the remark this answers is not in the document]
Comment (seungpark) [reply src/components/Card/index.js:74]: this is specifically for Card Groups with compact option. there is one on this example:

https://www.mongodb.com/docs/view-analyze/

https://docs-mongodbcom-integration.corp.mongodb.com/ca1cc8487251860566cc18198a0059a91c4d4fd7/master/landing/seung.park/DOP-2578-post-typography/view-analyze/

File: src/components/DriversIndexTiles.js [hunk 3]
```diff
@@ -102,15 +108,16 @@ const StyledGrid = styled('div')`
...
   flex-direction: row;
   padding: ${theme.size.default};
+  align-items: center;
+  border: 1px solid ${uiColors.gray.light2};
+  box-shadow: none;
 
   &:hover {
     box-shadow: 0 3px 6px -2px ${uiColors.gray.light1};
-    color: unset;
+    // color: unset;
```
[the remark this answers is not in the document]
Comment (seungpark) [reply src/components/DriversIndexTiles.js:120]: good catch! deleted

**Comment by seungpark** [comment]
> Screen Shot 2022-08-18 at 10 40 14 AM
> 
> Generally the cards look good, just had a few questions and small comments! I did have one note about this screenshot: should this "Still Learning MongoDB" card that shows when you scroll down the page also be styled similarly to the other Cards (currently it doesn't have the same rounded borders)?

removed the extra styling for rounded borders there. updating stage link

**Comment by seungpark** [comment]
some samples of changes with quizes and operations, with testing sample AST data in https://github.com/mongodb/snooty/tree/master/tests/unit/data

Screen Shot 2022-08-18 at 2 09 04 PM
Screen Shot 2022-08-18 at 2 09 30 PM
Screen Shot 2022-08-18 at 11 29 58 AM

**Review by allisonmui9 — APPROVED** [review]
LGTM thank you!

**Pull request merged**